core/src/main/java/hudson/init/impl/InstallUncaughtExceptionHandler.java

```diff
@@ -37,5 +40,33 @@ public void reportException(Throwable e, ServletContext context, HttpServletRequ
                 }
             }
         });
+        try {
+            Thread.setDefaultUncaughtExceptionHandler(new DefaultUncaughtExceptionHandler());
+            DefaultUncaughtExceptionHandler.LOGGER.log(Level.INFO, "Succesfully installed a global UncaughtExceptionHandler."); 
+        }
+        catch (SecurityException ex) {
```

Comment:
I think this is unnecessary. If there is a `SecurityManager` set, Jenkins will not run at all. It is designed to own the VM.